async-openai/src/fine_tune.rs

```diff
@@ -76,4 +76,14 @@ impl<'c, C: Config> FineTunes<'c, C> {
             )
             .await)
     }
+
+    /// Delete a fine-tuned model. You must have the Owner role in your organization.
+    pub async fn delete_fine_tun_model(
```

Comment:
This is actually present in `Models::delete` because of the REST endpoint being `/models/<something>` 

https://github.com/64bit/async-openai/blob/main/async-openai/src/model.rs#L33-L37

Reply:
fixed, remove it